src/lib/utils/index.ts

```diff
@@ -73,3 +73,19 @@ export const RUST_ISO_UTC_DATE_FORMAT = 'yyyy-MM-dd HH:mm:ss';
 export function isTouchScreenOnly(): boolean {
   return window.matchMedia('(hover: none) and (pointer: coarse)').matches;
 }
+
+export function getCookie(cname: string) {
+  let name = cname + '=';
+  let decodedCookie = decodeURIComponent(document.cookie);
+  let ca = decodedCookie.split(';');
+  for (let i = 0; i < ca.length; i++) {
+    let c = ca[i];
+    while (c.charAt(0) == ' ') {
+      c = c.substring(1);
+    }
+    if (c.indexOf(name) == 0) {
+      return c.substring(name.length, c.length);
+    }
+  }
+  return '';
```

Comment:
Dont love this fallback, as it implies (to me) that the cookie exists, but its value has been set to `''`. IMO better to either throw an error or return `undefined` (or `null`)

Reply:
agreed i think just returning undefined should have the desired effect